test/plugins/features/movies/controller.test.js

```diff
@@ -96,4 +114,19 @@ describe('movie controller', () => {
 
   });
 
+  describe('add_location', () => {
+
+    it('associates a location with a movie', () => {
```

Comment:
This test isn't actually checking that the location was added to the movie. You should be taking the newly fetched movie (with related) and making sure that `.related('locations')` has the location with id 1 in it.